### PR TITLE
basic app

### PR DESCRIPTION
**Pull request by Erich08**

### CODE REVIEW

**Comment by Dcastro99** [comment]
Like how you organized your code in different folders.  An idea I might steal!

**Comment by Jrdelt** [comment]
Clean usage of storing each component with its corresponding CSS files and passing data properly.

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,57 @@
...
+            className='map'
+            src={`https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_LOCATIONIQ_KEY}&center=${this.state.locationObj.lat},${this.state.locationObj.lon}&zoom=10`}
+            alt={this.state.locationObj.display_name}
+          />
+        )}
+        {this.state.locationObj.display_name && (
+          <p>
+            Latitude: {this.state.locationObj.lat} Longitude:{' '}
+            {this.state.locationObj.lon}
+          </p>
```
Comment (joshhaddock88) [inline src/App.js:50]: So this is a bit funny to me. Why not render your lat and lon inside your main? And why not render your image inside your main as well? Ideally youre App.js exclusively renders other components, and won't be rendering anything else.
So you App.js would have
```js
<div>
  <Main />
   <Search />
</div>
```
And then your main would render all the image, the p tags etc.

**Review by joshhaddock88 — COMMENTED** [review]
Don't forget to incorporate bootstrap! It can really help speed up the process of making things look pretty.

**Pull request merged**